poradnia/letters/templatetags/parse_markdown.py

```diff
@@ -0,0 +1,21 @@
+from django import template
+from django.template.defaultfilters import stringfilter
+from django.utils.safestring import mark_safe
+
+import markdown as md
+from bleach.sanitizer import Cleaner
+
+
+cleaner = Cleaner(
+tags=['a', 'abbr', 'acronym', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'ol', 'strong', 'ul', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'p', 'hr', 'pre', 'img'],
+attributes={'a': ['href', 'title'],'img': ['alt', 'src', 'title'] ,'abbr': ['title'], 'acronym': ['title']})
+
+register = template.Library()
+
+
+@register.filter()
+@stringfilter
+def markdown(value):
+    markdown = md.markdown(value, extensions=['markdown.extensions.fenced_code', 'sane_lists'])

```

Comment:
Jak już przyjmiemy, że nazwa jest bardziej generyczna to bym miał prośbę, aby dodać opcje konfiguracyjną dla formatowania. Nie wiem czy nie popsuje to historycznych wiadomości, więc chciałbym mieć możliwość łatwego wyłączenia.

Może wówczas nawet nie potrzebujemy od początku mieć bogatego edytora tekstowego, a damy użytkownikom podpowiedź co do składni. Na Wykopie i GitHubie to działa nawet.

W przyszłości istnieje szansa, że jeszcze dodatkowo będziemy formatować treść np. linkować sprawy na podstawie hasha.

Reply:
Co do opcji konfiguracyjnej - dobra myśl.
Rozumiem, że chodzi o prostą opcję, która by pozwoliła na przełączenie pomiędzy starym i nowym formatowaniem tekstu.

Reply:
Tak